creator/Documents/DifferencesBetweenBedrockAndJava.md

```diff
@@ -8,95 +8,133 @@ description: "An article covering the major differences between Minecraft: Bedro
 
-There are a lot of similarities between Bedrock Edition and Java Edition when it comes to resource packs but they do contain some differences. The obvious one is the swap of `.mcmeta` files with `.json` files in Bedrock Edition. These files are used to define the properties of different parts of the interface and are largely used in the same way with distinct syntax differences between the two formats. You’ll also see large differences in texture file formats, the main being the use of TGA files for work with alpha channels instead of PNG. Some textures (mainly entities) are laid out slightly different as well.
+## Resource Packs
+The idea behind resource packs is the same in both editions: change how the game looks. However, the capabilities and layout are very different.
 
-**Behavior Packs**
+Only Java Edition can change the shape of blocks, and only Bedrock Edition can change the shape of entities. Each uses a geometry format for this purpose that's incompatible with the other edition.
 
-One of the biggest differences between Bedrock Edition and Java Edition is the use of behavior packs. While functionally similar to the data packs in Java Edition, the actual implementation and use of behavior packs vary quite a bit.
+Many image files are named differently, since Java updated most of its names in 1.13 to be more consistent and Bedrock did not.
 
-Behavior packs bring in new functionality whether it be building your own entities from scratch, adding new blocks and items, or access to events using the JavaScript API. It allows for an enormous amount of flexibility and control and is one of the most powerful parts of Bedrock Edition when compared against Java Edition.
+To animate textures, Bedrock uses a single file called `flipbook_textures.json`, while Java uses individual `.mcmeta` files for each texture.
 
-**Gameplay and Player Input**
+Java Edition can create custom fonts and GLSL shaders, while Bedrock cannot. Bedrock Edition can create custom particles and fogs, while Java cannot.
 
-One major difference that tends to be forgotten is the type of platform players of different versions use. For Java Edition, you can be reasonably sure your player is using a keyboard and mouse; on Bedrock Edition, more than likely your player isn’t.
+## Behavior Packs
+Bedrock Edition's equivalent to Java's data packs are called behavior packs. Again, both share some similarities and some differences.
 
-Currently, console controls are the most common input method on Bedrock Edition with touch being a close second. Keyboard and mouse controls are a far third and make up a tiny percentage of your player base.
+One of Bedrock's most powerful and exclusive features is the ability to create entirely new custom entities with behavior packs. Custom items and blocks are also available in a limited, experimental form.
+
+Java Edition's data packs, meanwhile, allow for custom advancements, dimensions, and world generation.
 
```

Comment:
Same here, although there are few "it works differently" things to note. I mean, the schema for recipes and stuff is completely different for no reason, and Bedrock loot tables are missing about 50 functions, but that's not worth covering in a high-level overview